contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/actions.js

```diff
@@ -71,8 +71,8 @@ export function loadContentNodeByNodeId(context, nodeId) {
     });
 }
 
-export function loadChildren(context, { parent, published = null, complete = null }) {
-  const params = { parent, max_results: 25 };
+export function loadChildren(context, { parent, published = null, complete = null, ordering = 'lft' }) {
```

Comment:
I think we can leave this defaulting to `null` instead, so that we don't send an ordering parameter.